routes/markers.js

```diff
@@ -120,7 +120,7 @@ module.exports = function(connection) {
         var atlas_verison=req.body.atlas_version;
         var date_created= new Date().toISOString().slice(0, 19).replace('T', ' ');
         var date_changed=new Date().toISOString().slice(0, 19).replace('T', ' ');
-        var created_by=req.body.uid;
+        var created_by=req.session.user.uid;
```

Comment:
If we're creating a new marker, `created_by` is the current user; however, if we're updating a marker, then`created_by` should not be in the list of attributes we are updating.